packages/grafana-ui/src/components/OptionsUI/mappings.tsx

```diff
@@ -11,6 +16,12 @@ export class ValueMappingsValueEditor extends React.PureComponent<
   }
 
   render() {
+    deprecationWarning(
+      'mappings.tsx',
+      'grafana-ui/src/components/OptionsUI',
+      'public/app/features/dimensions/editors/ValueMappingsEditor'
+    );
```

Comment:
this deprecation warning does not make much sense as plugins cannot use components in the app. I would just remove it